.test/oci-sort-platforms/test.jq

```diff
@@ -0,0 +1,72 @@
+include "oci";
+
+[
+	{
+		os: "linux",
+		architecture: (
+			"386",
+			"amd64",
+			"arm",
+			"arm64",
+			"mips64le",
+			"ppc64le",
+			"riscv64",
+			"s390x",
+			empty
+		),
+	},
+
+	{
+		os: "windows",
+		architecture: ( "amd64", "arm64" ),
+		"os.version": (
+			# https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle
+			# https://oci.dag.dev/?repo=mcr.microsoft.com/windows/servercore
+			# https://oci.dag.dev/?image=hell/win:core
+			"10.0.14393.6796",
+			"10.0.16299.1087",
+			"10.0.17134.1305",
+			"10.0.17763.5576",
+			"10.0.18362.1256",
+			"10.0.18363.1556",
+			"10.0.19041.1415",
+			"10.0.19042.1889",
+			"10.0.20348.2340",
+			empty
+		)
+	},
+
+	{
+		os: "freebsd",
+		architecture: ( "amd64", "arm64" ),
+		"os.version": ( "12.1", "13.1" ),
+	},
+
+	# buildkit attestations
+	# https://github.com/moby/buildkit/blob/5e0fe2793d529209ad52e811129f644d972ea094/docs/attestations/attestation-storage.md#attestation-manifest-descriptor
+	{
+		architecture: "unknown",
+		os: "unknown",
+	},
+
+	empty
```

Comment:
Smaller diffs in the future -- it allows a "list" to end with a trailing comma (which is otherwise invalid syntax) so that we don't have to modify the previous item to add a new one :+1:

Reply:
(sometimes I write this as `empty # trailing comma hack` but it's a little tedious to do so consistently :sweat_smile:)